sensor/common/detector/baseline/baseline.go

```diff
@@ -47,6 +52,7 @@ func (w *baselineEvaluator) AddBaseline(baseline *storage.ProcessBaseline) {
 		defer w.baselineLock.Unlock()
 
 		delete(w.baselines[baseline.GetKey().GetDeploymentId()], baseline.GetKey().GetContainerName())
+		log.Infof("Deleted baseline %s", baseline.GetId())
```

Comment:
[nit] I'd add here `process baseline` to not confuse it with the network baseline.

Reply:
done